sympy/parsing/tests/test_autolev.py

```diff
@@ -11,36 +13,55 @@ def _test_examples(in_filename, out_filename, test_name=""):
     from sympy.parsing.autolev import parse_autolev
 
     dir_path = os.path.dirname(os.path.dirname(os.path.abspath(os.path.realpath(__file__))))
-    in_file_path = os.path.join(dir_path, 'autolev', 'test_examples', in_filename)
-    correct_file_path = os.path.join(dir_path, 'autolev', 'test_examples', out_filename)
-    out_file_path = os.path.join(dir_path, 'autolev', 'test_examples', 'output.py')
+    in_file_path = os.path.join(dir_path, 'autolev', 'test-examples', in_filename)
+    correct_file_path = os.path.join(dir_path, 'autolev', 'test-examples', out_filename)
+    with open(in_file_path) as f:
+        generated_code = parse_autolev(f, include_numeric=True)
 
-    parse_autolev(in_file_path, out_file_path)
-    #parse_autolev(in_file_path, correct_file_path)
-    with open(out_file_path) as f1, open(correct_file_path) as f2:
-
-            for idx, (lineA, lineB) in enumerate(zip(f1, f2)):
-                try:
-                    assert lineA.rstrip() == lineB.rstrip()
-                except Exception:
-                    raise AssertionError('mismatch in ' + test_name + ' in line no: {0}'.format(idx+1))
+    with open(correct_file_path) as f:
+        for idx, line1 in enumerate(f):
+            if line1.startswith("#"):
+                break
+            try:
+               line2 = generated_code.split('\n')[idx]
+               assert line1.rstrip() == line2.rstrip()
+            except Exception:
+                raise AssertionError('mismatch in ' + test_name + ' in line no: {0}'.format(idx+1))
 
 
 def test_rule_tests():
-    number_of_tests = 12
-    for i in range(1, number_of_tests+1):
-        in_filepath = "ruletest" + str(i) + ".al"
-        out_filepath = "ruletest" + str(i) + ".py"
-        _test_examples(in_filepath, out_filepath, "test_" + str(i))
+    l = ["ruletest1", "ruletest2", "ruletest3", "ruletest4", "ruletest5", "ruletest6",\
+         "ruletest7", "ruletest8", "ruletest9", "ruletest10", "ruletest11", "ruletest12"]
+    for i in l:
+        in_filepath = i + ".al"
+        out_filepath = i + ".py"
+        _test_examples(in_filepath, out_filepath, i)
+
+
+def test_pydy_examples():
+    l = ["mass_spring_damper", "chaos_pendulum", "double_pendulum", "non_min_pendulum"]
+    for i in l:
+        in_filepath = os.path.join("pydy-example-repo", i + ".al")
+        out_filepath = os.path.join("pydy-example-repo", i + ".py")
+        _test_examples(in_filepath, out_filepath, i)
 
-def test_mass_spring_damper():
-    _test_examples("mass_spring_damper.al", "mass_spring_damper.py")
 
-def test_chaos_pendulum():
-    _test_examples("chaos_pendulum.al", "chaos_pendulum.py")
+# def test_autolev_tutorial():
+#     l = ["tutor1", "tutor2", "tutor3", "tutor4", "tutor5", "tutor6", "tutor7"]
+#     for i in l:
+#         in_filepath = os.path.join("autolev-tutorial", i + ".al")
+#         out_filepath = os.path.join("autolev-tutorial", i + ".py")
+#         _test_examples(in_filepath, out_filepath, i)
```

Comment:
Don't leave commented code in like this.

Reply:
I left these like this because the autolev-tutorial and dynamics-online directories can be copied over from the GitLab repo and tested by uncommenting this code.

Reply:
I see. It would be better to have them uncommented and use an if statement that runs them if the files are there, but doesn't if the files aren't.

Reply:
That sounds better. I shall change it.